lib/modules/manager/bazel-module/bazelrc.ts

```diff
@@ -0,0 +1,140 @@
+import upath from 'upath';
+import { isNotNullOrUndefined } from '../../../util/array';
+import * as fs from '../../../util/fs';
+import { regEx } from '../../../util/regex';
+
+const importRegex = regEx(`^(?<type>(?:try-)?import)\\s+(?<path>\\S+)$`);
+const optionRegex = regEx(
+  `^(?<command>\\w+)(:(?<config>\\S+))?\\s+(?<options>.*)$`
+);
+const spaceRegex = regEx(`\\s+`);
+
+export class ImportEntry {
+  readonly entryType = 'import';
+  constructor(readonly path: string, readonly isTry: boolean) {}
+}
+
+export class BazelOption {
+  constructor(readonly name: string, readonly value?: string) {}
+
+  static parse(input: string): BazelOption[] {
+    const options: BazelOption[] = [];
+    const parts = input.split(spaceRegex);
+    for (let i = 0; i < parts.length; i++) {
+      const part = parts[i];
+      if (part.startsWith('--')) {
+        const nameStartIdx = 2;
+        let name: string;
+        let value: string | undefined;
+        // Check for --option_name=option_value
+        const equalSignIdx = part.indexOf('=');
+        if (equalSignIdx >= 0) {
+          name = part.substring(nameStartIdx, equalSignIdx);
+          value = part.substring(equalSignIdx + 1);
+        } else {
+          name = part.substring(nameStartIdx);
+          const nextIdx = i + 1;
+          // Check for --option_name OR --option_name option_value
+          value =
+            nextIdx < parts.length && !parts[nextIdx].startsWith('--')
+              ? parts[nextIdx]
+              : undefined;
+        }
+        const option = new BazelOption(name, value);
+        options.push(option);
+      }
+    }
```

Comment:
```suggestion
    for (let i = 0; i < parts.length; i++) {
      const part = parts[i];
      if (!part.startsWith('--')) {
        continue;
      }

      const nameStartIdx = 2;
      // Check for --option_name=option_value
      const equalSignIdx = part.indexOf('=');
      if (equalSignIdx >= 0) {
        const name = part.substring(nameStartIdx, equalSignIdx);
        const value = part.substring(equalSignIdx + 1);
        options.push(new BazelOption(name, value));
        continue;
      }

      // Check for --option_name OR --option_name option_value
      const name = part.substring(nameStartIdx);
      const nextIdx = i + 1;
      const value =
        nextIdx < parts.length && !parts[nextIdx].startsWith('--')
          ? parts[nextIdx]
          : undefined;
      options.push(new BazelOption(name, value));
    }
```
Use early returns for easier readability.